phpunit.xml.dist

```diff
@@ -1,23 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
```

Comment:
do we require to use the online schema. Would keep it with the local schema as it represent always the current version which is installed?

Reply:
For the root directory this might be true but the individual modules have the site linked.

Reply:
make sense